lib/main_containers.pm

```diff
@@ -140,7 +140,7 @@ sub load_host_tests_docker {
         # to maintenance jobs or new products after Beta release
         # PackageHub is not available in SLE Micro | MicroOS
         loadtest 'containers/registry' if (is_x86_64 || is_sle('>=15-sp4'));
-        loadtest 'containers/docker_compose' unless is_public_cloud;
+        loadtest 'containers/docker_compose' unless (is_public_cloud || is_sle('=12-sp3'));
```

Comment:
in which versions we DO want to run it ? is it ok if it will run on 12-sp4 ? 12-sp5 ?

Reply:
12sp4 has been removed, and it works fine in 12sp5.

Reply:
```suggestion
        loadtest 'containers/docker_compose' unless (is_public_cloud || is_sle('<12-sp5'));
```

just to be on the safe side , who knows maybe one day company which is the reason why we need to run 12 sp3 will switch to .... 12 sp4 :D